english/dimensions/dimension.md

```diff
@@ -2,6 +2,8 @@ And here you enter a new [dimension](https://www.youtube.com/watch?v=sea8yNCDqWo
 
 Or decide to play a game of chess [chess](../chess/chess.md).
 
+or would you like to challenge Yami Yugi and [dimension the dice!](../Yu-Gi-Oh/yami%20yugi.md)
+
 Or decide to learn about the world of bears [bears](../bear-story/bear-story.md)
 
-Or decide the learn about life of this world [life of this world](../00-Life-of-this-world/Life of this world.md)
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.